convert/jenkinsjson/json/artifactoryRtCommands.go

```diff
@@ -0,0 +1,272 @@
+package json
+
+import (
+	"errors"
+	"fmt"
+	harness "github.com/drone/spec/dist/go"
+)
+
+const (
+	ArtifactoryRtCommandsPluginImage = "plugins/artifactory"
+	InputPlaceHolder                 = "<+input>"
+	MvnTool                          = "mvn"
+	GradleTool                       = "gradle"
+)
+
+func ConvertArtifactoryRtCommand(stepType string, node Node, variables map[string]string) *harness.Step {
+	switch stepType {
+	case "rtDownload":
+		return convertRtDownload(node, variables)
+	case "rtMavenRun":
+		return convertRtMavenRun(node, variables)
+	case "rtGradleRun":
+		return convertRtGradleRun(node, variables)
+	case "publishBuildInfo":
+		return convertPublishBuildInfo(node, variables)
+	case "rtPromote":
+		return convertRtPromote(node, variables)
+	case "xrayScan":
+		return convertXrayScan(node, variables)
+	}
+	return nil
+}
+
+var ConvertRtMavenRunParamMapperList = []JenkinsToDroneParamMapper{
+	{"pom", "source", StringType, nil},
+	{"goals", "goals", StringType, nil},
+	{"buildName", "build_name", StringType, nil},
+	{"buildNumber", "build_number", StringType, nil},
+}
+
+func convertRtMavenRun(node Node, variables map[string]string) *harness.Step {
+	step := GetStepWithProperties(&node, ConvertRtMavenRunParamMapperList, ArtifactoryRtCommandsPluginImage)
+	if step == nil {
+		fmt.Println("Error: failed to convert rtMavenRun")
+		return nil
+	}
+	tmpStepPlugin, ok := step.Spec.(*harness.StepPlugin)
+	if !ok {
+		fmt.Println("Error: failed to convert rtMavenRun")
+		return nil
+	}
+	if tmpStepPlugin.With == nil {
+		tmpStepPlugin.With = map[string]interface{}{}
+	}
+	tmpStepPlugin.With["build_tool"] = MvnTool
+	attributesList := []string{"url", "username", "password", "access_token",
```

Comment:
remove `password` from all the steps and only keep `access_token` - also, update the test cases